### ISSUE TITLE
Add basic social media icons

### ISSUE THREAD

**Issue by patrickarlt** [Esri/calcite-ui-icons#229]
It would be nice to have some standardized icons for:

* Facebook
* GitHub
* Twitter
* Google
* Apple

As a part of this project. Right now we have to distribute these ourselves for the developers website.

**Comment by KatelynSeitz** [comment]
Hey @patrickarlt 
Right now standardized assets for 3rd party logos can be found [here in AEM](https://aem-author-prd.esri.com/assets.html/content/dam/esrisites/en-us/common/icons/social-media/svgs). The reason is because we want to follow the logo guidelines of these brands instead of styling them under Calcite UI icons.

**Comment by patrickarlt** [comment]
@SkyeSeitz I don't have access to those resources. That was kind of the main point of this issue is that distributing those resources in a centralized way would be great as opposed to letting each team sort it out for themselves from whatever resources they can find online.

**Comment by SkyeSeitz** [comment]
Good point. AEM is a centralized location for Brand (Marketing) teams to get and implement what they need, but a similar resource is needed for Product teams. If Calcite UI Icons provided filled versions of the logos above would not violate the brand guidelines of those companies. Calcite UI Icons is an outlined style, but I think you are right it is the most appropriate location for product teams to access them.
